README.md

```diff
@@ -15,9 +15,27 @@ from scratch as well as keep it up-to-date over time.
 
 ### Prerequisites
 
+* `brew`
+
+```bash
+xcode-select -–install
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+>see: [Install Homebrew without XCODE](https://www.codeandpeace.com/install-homebrew-without-xcode/)
+
 * `git` must be installed
+
+```bash
+brew install git
+```
+
 * `bash` should be configured as the preferred shell
 
+```bash
```

Comment:
this would only apply on systems where bash is not already default (if they changed it i assume they can change it back...)